### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by mbelluzzo**
Glib says hi to conffile!

Coverity Issue 126578 should be gone.

And tomorrow I'm sure coverity will surprise us again :-)

### CODE REVIEW

File: src/lib/flow/Kconfig [hunk 1]
```diff
@@ -13,7 +13,7 @@ config NODE_DESCRIPTION
 
 config RESOLVER_CONFFILE
 	bool "Resolver conffile"
-	depends on NODE_DESCRIPTION
+	depends on NODE_DESCRIPTION && HAVE_GLIB
```
Comment (lpereira) [inline src/lib/flow/Kconfig]: Why?

Comment (mbelluzzo) [inline src/lib/flow/Kconfig]: Because resolver-conffile still depends on glib :-(

Comment (ulissesf) [inline src/lib/flow/Kconfig]: Ok, but can't we then submit a patch to remove the dependency instead of adding it back again?

Comment (mbelluzzo) [inline src/lib/flow/Kconfig]: Why have the build broken until then? The patch should fix resolver-conffile and remove this line. Isn't an one line patch and is breaking galileo build that is being used by QA

**Pull request closed without merging**